src/figuresClasses.ts

```diff
@@ -1,11 +1,80 @@
-export interface Figure {}
+import { get } from 'http';
 
-export class Triangle implements Figure {}
+interface Figure {
+  getArea(): number;
+  getInfo(): string;
+}
+
+class Triangle implements Figure {
+  constructor(
+    private color: string,
+    private a: number,
+    private b: number,
+    private c: number,
```

Comment:
[CODE KNOWLEDGE] - You don't need to manually assign constructor parameters to class properties if you use access modifiers like 'private'. TypeScript does this automatically. You can remove the assignments and leave the constructor body empty.